regulation-worker/internal/delete/api/api.go

```diff
@@ -24,7 +24,7 @@ import (
 
 var (
 	pkgLogger             = logger.NewLogger().Child("api")
-	supportedDestinations = []string{"BRAZE", "AM", "INTERCOM", "CLEVERTAP", "AF", "MP", "GA"}
+	supportedDestinations = []string{"BRAZE", "AM", "INTERCOM", "CLEVERTAP", "AF", "MP", "GA", "ITERABLE"}
```

Comment:
Is there any other destination we support user deletion but not on this list?

Reply:
yes, engage, custify and sendgrid

Reply:
Let's add these as well.